adafruit_mcp4728.py

```diff
@@ -232,12 +241,12 @@ def _generate_bytes_with_flags(channel):
         return buf
 
     @staticmethod
-    def _chunk(big_list, chunk_size):
+    def _chunk(big_list: bytearray, chunk_size: int) -> bytearray:
```

Comment:
Oh, a generator!  We can type the use of `yield` using `typing.Generator`, and for less complex generators like this one, we can actually use `typing.Iterator`:

```suggestion
    def _chunk(big_list: bytearray, chunk_size: int) -> Iterator[bytearray]:
```